programs/editor/plugins/bella/BellaControl.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (C) 2014-2015 KO GmbH <copyright@kogmbh.com>
+ *
+ * @licstart
+ * This file is part of WebODF.
+ *
+ * WebODF is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License (GNU AGPL)
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * WebODF is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with WebODF.  If not, see <http://www.gnu.org/licenses/>.
+ * @licend
+ *
+ * @source: http://www.webodf.org/
+ * @source: https://github.com/kogmbh/WebODF/
+ */
+
+/*global define, runtime*/
+
+define("webodf/editor/plugins/bella/BellaControl", [
+        "webodf/editor/plugins/bella/Bella", "dijit/form/Button"
+    ],
+    function(Bella, Button) {
+    "use strict";
+
+    function BellaControl(config, onInitialise) {
+        var self = this,
+            bella,
+            editorSession,
+            startButton,
+            stopButton,
+            widget = {};
+
+        startButton = new Button({
+            label: runtime.tr('Start'),
```

Comment:
"Start Testing" / "Stop Testing" might become too large, at least with the splitscreen version it fills small screens up too much IMHO.
Sadly I found no nice matching icons in the default Dojo icon set, otherwise I would have taken those icons and used more explicite text as tooltips here.
Argument to myself was also that for now there are no other "Start" & "Stop" actions known that might conflict. And usually this plugin is activated only by people who know what to expect.
Can you agree to that argument? :)